README.md

```diff
@@ -1,5 +1,14 @@
 # rhof
 
+[![Build](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml/badge.svg)](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml)
+[![NPM Version](https://img.shields.io/npm/v/rhof.svg)](https://www.npmjs.com/package/rhof)
+[![NPM Downloads](https://img.shields.io/npm/dt/rhof.svg)](https://www.npmjs.com/package/rhof)
+[![Github Repo Size](https://img.shields.io/github/repo-size/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof)
+[![LICENSE](https://img.shields.io/npm/l/rhof.svg)](https://github.com/pwnb0y/rhof/blob/master/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof/graphs/contributors)
+[![Commit](https://img.shields.io/github/last-commit/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof/commits/master)
+
+
 <!-- Add Badges here -->
```

Comment:
Can we also remove these from line 3 to 9 since we are using badges added by npm-badger now

Reply:
Remove these lines https://github.com/gh-conf/rhof/pull/50/commits/916dc6db699a2321cc12141443e5aad7d223208f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R12-R18

Reply:
@pwnb0y this is still pending. We can remove line 12 - 18 in README.md